webapp/src/selectors.ts

```diff
@@ -358,6 +356,15 @@ export const hasPermissionsToEnableCalls = (state: GlobalState, channelId: strin
 //
 // Selectors for Cloud and beta limits:
 //
+// Having trouble importing this, so embed.
+enum LicenseSkus {
```

Comment:
I gave up here, I'm not sure why it's not importing correctly. Maybe I'll try to figure it out again later.